docs/source/DeveloperDocumentation/PackagingGuides/aws.md

```diff
@@ -1,9 +1,60 @@
-# SPEKE KEY PROVIDER API
-These are the docs for the speke key provider, which has a speke endpoint for use with AWS.
+# AWS Media Services
 
 In order to use our SPEKE API you will need to set up an API gateway that adds the header `API-KEY` with your API key and with your client name in place of `client-name` in "https://speke.vudrm.tech/client-name/speke". For more information on how to setup the API Gateway see the [API Gateway](#api-gateway) section.
 
-## Use with AWS Media Convert (VOD)
+## Media Packager
+### 1. Go to AWS Media Packager.
+-	Log into the AWS console
+-	Either type “Media Packager” into the search box and then select “MediaPackager” or open the “Services” drop down in the top left of the screen and select “MediaPackager” from under the “Media Services” header.
+
+### 2. Select or create a channel.
+-   To create a channel press the "Create" button in the top right corner, enter an ID for the channel and press create.
+
+### 3. Add an endpoint to the channel.
+-   Press the "Add/edit endpoints" button
+-   Press the "Add" button to create a new endpoint
+
+### 4. Assign an ID to the endpoint.
+-   You can also add a description of the endpoint if needed
+
+### 5. Set the "Manifest Name" to "manifest".
+
+### 6. Set the type of stream.
+-   Use DASH-ISO if you want to use Playready or Widevine DRM
+-   Use Apple HLS if you want to use Fairplay DRM
+
+### 7. Select the option to "Encrypt content".
+
+### 8. Add a Resource ID.
+
+### 9. Add appropriate System IDs.
+-   If you wish to use Widevine DRM use "edef8ba9-79d6-4ace-a3c8-27dcd51d21ed" as the System ID (only if output group is DASH ISO)
+-   If you wish to use Playready DRM use "9a04f079-9840-4286-ab92-e65be0885f95" as the System ID (only if output group is DASH ISO)
+-   If you wish to use Fairplay DRM use "94ce86fb-07ff-4f43-adb8-93d2fa968ca2" as the System ID (only if output group is APPLE HLS)
+
+### 10. Set the URL to that of your api gateway.
+
+### 11. Add an appropriate Role ARN.
+
+### 12. Press the "Save" button under the list of endpoint. 
+
+### 13. To test the content, copy the endpoints URL.
+
+### 14. Load into a player to show the content working.
+#### Widevine
+-	Open “https://cdn.vuplay.co.uk/vuplay-smart-demo/_latest/index.html#/” in a new Google Chrome tab and select custom stream.
```

Comment:
Should probably point towards the test player in the admin site instead of vuplay.

Reply:
It should point to the test player - is that possible though, since each customer has their own URL? Maybe just point to the Admin site dashboard page and mention the 'Test your stream' tab.